tests/test_parse.py

```diff
@@ -49,30 +48,31 @@ def test_simple_emr_log():
 
     parsed = get_parsed_log(event_log_path)
 
-    assert all(
-        key in parsed
-        for key in [
-            "accumData",
-            "executors",
-            "jobData",
-            "metadata",
-            "sqlData",
-            "stageData",
-            "taskData",
-        ]
-    ), "Not all keys are present"
+    assert all(key in parsed for key in PARSED_KEYS), "Not all keys are present"
 
     assert (
         parsed["metadata"]["application_info"]["name"] == "Text Similarity"
     ), "Name is as expected"
 
 
+def test_parsed_log():
+    event_log_path = Path("tests", "logs", "similarity_parsed.json.gz").resolve()
+
+    with tempfile.TemporaryDirectory() as temp_dir:
+        event_log_paths = extractor.Extractor(event_log_path.resolve().as_uri(), temp_dir).extract()
+        event_log, parsed = eventlog.EventLogBuilder(event_log_paths, temp_dir).build()
+        sparkApplication(spark_eventlog_parsed_path=str(event_log))
+
+    assert parsed
```

Comment:
I think this would be slightly easier to follow if the assert immediately followed the line in which `parsed` is assigned.